pkg/index/main.go

```diff
@@ -84,8 +84,8 @@ func Create(config *config.Options) error {
 		//fmt.Printf("found release %s\n", *r.TagName)
 		var packageName, packageVersion, packageURL string
 		for _, f := range r.Assets {
-			tagParts := strings.Split(*r.TagName, "+")
-			if len(tagParts) == 2 && *f.Name == fmt.Sprintf("%s-%s.tgz", tagParts[1], tagParts[0]) {
+			tagParts := strings.Split(*r.TagName, "-")
```

Comment:
I think we should split at the last index of `-`, otherwise for a chart named chart named e.g. `foo-bar` hence the tag name being `foo-bar-1.0.0`, we'll get `{"foo", "bar-1.0.0"}` which isn't what we want.